### PR TITLE
Remove nuimo

### PR DESCRIPTION
**Pull request by rngtng**
## Proposed change
Remove Nuimo Controller Documentation

As of https://github.com/home-assistant/core/pull/45600 Nuimo controller integration is deleted.

## Type of change
Deletion of existing documentation

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45600
- Link to parent pull request in the Brands repository: -
- This PR fixes or closes issue: -

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ]  (not applicable) The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
Please remove all references to the page as well, and add a redirect to the redirect file (more removed integrations can be found in there) to handle people still landing on this page.

**Comment by rngtng** [comment]
@frenck done

File: source/_posts/2019-09-18-release-99.markdown [hunk 1]
```diff
@@ -107,7 +107,7 @@ Home Assistant had a booth this year at the [CEDIA home tech trade show](https:/
 
 <blockquote class="twitter-tweet"><p lang="en" dir="ltr">Hanging out at <a href="https://twitter.com/CEDIAExpo?ref_src=twsrc%5Etfw">@CEDIAExpo</a> in innovation alley <a href="https://t.co/9SnUXo11oA">pic.twitter.com/9SnUXo11oA</a></p>&mdash; Home Assistant (@home_assistant) <a href="https://twitter.com/home_assistant/status/1172559987979292673?ref_src=twsrc%5Etfw">September 13, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 
-We also had some great drinks in the evenings with companies that integrate with Home Assistant. In this picture [Konnected.io](/integrations/konnected) and [Senic](/integrations/nuimo_controller/).
+We also had some great drinks in the evenings with companies that integrate with Home Assistant. In this picture [Konnected.io](/integrations/konnected) and Senic
```
Comment (frenck) [inline source/_posts/2019-09-18-release-99.markdown]:
```suggestion
We also had some great drinks in the evenings with companies that integrate with Home Assistant. In this picture [Konnected.io](/integrations/konnected) and Senic.
```

File: source/_posts/2017-05-06-zigbee-opencv-dlib.markdown [hunk 2]
```diff
@@ -183,7 +183,7 @@ influxdb:
 - Remove path whitelisting for hassio ([@balloob] - [#7399])
 - Add Eight sleep component ([@mezz64] - [#7275]) ([eight_sleep docs]) ([binary_sensor.eight_sleep docs]) ([sensor.eight_sleep docs]) (new-platform)
 - OpenGarage support ([@swbradshaw] - [#7338]) ([cover.opengarage docs]) (new-platform)
-- Clean up requirements ([@andrey-git] - [#7391]) ([nuimo_controller docs]) ([sensor.dht docs]) ([sensor.thinkingcleaner docs]) ([switch.thinkingcleaner docs])
+- Clean up requirements ([@andrey-git] - [#7391]) (nuimo_controller docs) ([sensor.dht docs]) ([sensor.thinkingcleaner docs]) ([switch.thinkingcleaner docs])
```
Comment (frenck) [inline source/_posts/2017-05-06-zigbee-opencv-dlib.markdown]:
```suggestion
- Clean up requirements ([@andrey-git] - [#7391]) ([sensor.dht docs]) ([sensor.thinkingcleaner docs]) ([switch.thinkingcleaner docs])
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @rngtng 👍

**Pull request merged**